tools/cli/commands/build.js

```diff
@@ -919,10 +920,21 @@ async function buildProject( t ) {
 		crlfDelay: Infinity,
 	} );
 
-	rl.on( 'line', line => {
+	rl.on( 'line', async line => {
```

Comment:
Using `async` here doesn't work right. When the `await` happens `rl` continues processing lines, which can result in a later version line of the changelog getting processed, incorrectly overwriting `projectVersionNumber` with a wrong version before the `git` command finishes.

Probably the best thing to do here would be to move lines 927–937 outside of the event handler entirely, probably to around what's currently line 947.